tests/src/test/scala/org/http4s/parser/ContentBaseHeaderSpec.scala

```diff
@@ -0,0 +1,32 @@
+package org.http4s
+package parser
+
+import org.http4s.headers.`Content-Base`
+
+class ContentBaseHeaderSpec extends Http4sSpec {
+  "ContentBaseHeader parser".can {
+    "Parse a simple uri" in {
+      val s = "http://www.foo.com"
+      val Right(uri) = Uri.fromString(s)
+      val hs = Headers.of(Header("Content-Base", s))
+
+      hs.get(`Content-Base`) must beSome(`Content-Base`(uri))
+    }
+
+    "Parse a simple uri with a path but no authority" in {
+      val s = "http:/foo/bar"
+      val Right(uri) = Uri.fromString(s)
+      val hs = Headers.of(Header("Content-Base", s))
+
+      hs.get(`Content-Base`) must beSome(`Content-Base`(uri))
+    }
+
+    "Parse a relative reference" in {
```

Comment:
I think this one is actually invalid?  The spec says it has to be an absolute URI.  Which is unfortunate, because we don't have that at the type level, so it will be easy to create an invalid one even if the parser rejects this.  Any ideas?